server/sonar-db-dao/src/main/resources/org/sonar/db/qualityprofile/QualityProfileMapper.xml

```diff
@@ -237,6 +239,8 @@
       <foreach collection="languages" open="(" close=")" item="language" separator=",">
         #{language}
       </foreach>
+      AND p.organization_uuid = #{organizationUuid}
```

Comment:
jdbcType is missing from this commit. They are incorrecly added in the unrelated next commit